soundex/__init__.py

```diff
@@ -142,12 +142,15 @@ def compare(self, string1, string2):
         soundex1 = self.soundex(string1)
         soundex2 = self.soundex(string2)
 
-        if soundex1[1:] == soundex2[1:]:
+        if soundex1[1:] == soundex2[1:] and soundex1[0] == soundex2[0]:
             # Strings sound phonetically same
             return 1
 
         # Strings are not same
-        return 2
+        if soundex1[1:] == soundex2[1:] and soundex1[0] != soundex2[0]:
+            return 2
+        else:
+            return -1
```

Comment:
Return 2 , phonetically same but different languages. I have explained as comments in the new commits.